github/ApplicationOAuth.pyi

```diff
@@ -13,12 +13,12 @@ class ApplicationOAuth(NonCompletableGithubObject):
     def client_secret(self) -> str: ...
     def get_login_url(
         self,
-        redirect_uri: Optional[str],
-        state: Optional[str],
-        login: Optional[str]
+        redirect_uri: Optional[str]=None,
```

Comment:
`.pyi` files usually do not have default values but `...` instead, default values are defined in the respective `.py` files, same issue in other files

Reply:
Sorry about those, I had never used pyi files before, so I'm used to adding the hints where the default value is. Should all be fixed now.